README.md

```diff
@@ -1,40 +1,60 @@
 # Core APIs NestJS Service Template
 
 MadeiraMadeira boilerplate project to build scalable, testable and high performance Node.js microservices. 
- 
+
+## Service Architecture
+This service make part of the project <project name>.
+![Swagger OpenApi](docs/service-arch.png)
+> Obs: image to describe the service architecture or project architecture.
+
+## General Kong Routes Architecture
+Kong routes map.
+
+![Swagger OpenApi](docs/service-routes.png)
+> Obs: image to describe the service routes or project routes.
+
 ## Stack
 
 - [NestJS](https://github.com/nestjs/nest): a progressive Node.js + TypeScript framework based on dependency injection;
 - [gRPC](https://grpc.io): a Remote Procedure Call framework that provides high performance communication between microservices.
+- [Swagger + OpenAPI](https://swagger.io/specification/): an OpenAPI definition can then be used by documentation generation tools to display the API, code generation tools to generate servers and clients in various programming languages, testing tools, and many other use cases. 
+
+## Swagger Docs
+The current documentation of the project can be found on url `/docs`.
+![Swagger OpenApi](docs/openapi.png)
+
+## Release notes
+In this section contains the release notes of the project.
+
+> Version 1.0.0
+
+First version of the project;
+All the changes must be tracked in [CHANGELOG.md](CHANGELOG.md)
+
+## Prerequisites
+- Node 12
 
 ## Features
-- REST and gRPC layers; 
+- REST and gRPC layers;
+- Swagger; 
 - Husky Git hooks that automatically runs [ESLint](https://eslint.org) and [Prettier](https://prettier.io) before all commits;
 - `ConfigService`: easily manage environment variables;
 - `Health` module: a gRPC and a REST endpoint that displays relevant information about the application status;
 - Docker infrastructure with Docker Compose.
 
-## Installation for local development with Docker
-
-### Installation
+## Installation
+### Preparing the environment
+Copy the sample env file using the follow command:
 ```bash
-$ cp example.env .env
-$ npm install --target=12.0.0 --target_platform=linux --target_arch=x64 --target_libc=musl
+cp example.env .env
 ```
-Notice: these flags are needed to build the gRPC package to the correct container architecture. This issue
-will be fixed in the next major release of NestJS (https://github.com/nestjs/nest/pull/6349). 
 
-### Running the app
+### Running locally
```

Comment:
I think you should replace this with "Installing Locally" or something. The "Running" section is right below, and is currently duplicated, which may cause confusion.